ansible/roles/kubernetes/defaults/main.yml

```diff
@@ -37,14 +37,15 @@ kube_current_release_directory: "{{ kube_releases_directory }}/{{ kube_version }
 kube_cert_dir: "{{ kube_config_dir }}/certs"
 
 # The IP(s) for which the certificate will be valid
-kube_cert_ip: "{{ hostvars[inventory_hostname]['ansible_all_ipv4_addresses'] | join (',IP:') }}"
+kube_cert_ip: "{{ (hostvars[inventory_hostname]['ansible_all_ipv4_addresses'] | join (',IP:'))}}{% if kube_apiserver_vip %},IP:{{ kube_apiserver_vip }}{% endif %}"
 
 # This is where all of the bearer tokens will be stored
 kube_token_dir: "{{ kube_config_dir }}/tokens"
 
 # This is where you can drop yaml/json files and the kubelet will run those
 # pods on startup
 kube_manifest_dir: "{{ kube_config_dir }}/manifests"
+kube_standby_manifest_dir: "{{ kube_config_dir }}/standby_manifests"
```

Comment:
Can you add a space and an inline comment that describes the param? Can kube_manifest_dir simply be used or is a separate standby_manifests directory required?